src/EventListener/CreateSchemaListener.php

```diff
@@ -123,7 +123,11 @@ public function postGenerateSchemaTable(GenerateSchemaTableEventArgs $eventArgs)
             /** @var string[] $foreignColumnNames */
             $foreignColumnNames = array_keys($keyColumns);
         } else {
-            /** @var string[] $foreignColumnNames */
+            /**
+             * @phpstan-ignore-next-line
```

Comment:
This is needed because of the new lie detector of phpstan. Since we are supporting multiple versions of dbal.

```
 ------ ----------------------------------------------------------------------------------------------------
  Line   src/EventListener/CreateSchemaListener.php
 ------ ----------------------------------------------------------------------------------------------------
  129    PHPDoc tag @var with type array<string> is not subtype of type array<Doctrine\DBAL\Schema\Column>.
 ------ ----------------------------------------------------------------------------------------------------
```